src/DynamoCore/Core/Threading/UpdateGraphAsyncTask.cs

```diff
@@ -150,15 +150,20 @@ private bool Contains(UpdateGraphAsyncTask other)
                    other.graphSyncData.DeletedNodeIDs.All(graphSyncData.DeletedNodeIDs.Contains);
         }
 
+        private bool IsAfter(UpdateGraphAsyncTask other)
+        {
+            return this.CreationTime > other.CreationTime;
+        }
+
```

Comment:
This is okay, but I am wondering if it can be called `IsScheduledAfter` to make the method more self-explanatory?